main.py

```diff
@@ -137,6 +137,9 @@ async def persist_smt(self, smt, gameId):
       time.sleep(0.3)
       plugin_utils.set_power_governor_for_tdp_profile(tdp_profile)
 
+  async def set_smt(self, smt):
+      cpu_utils.set_smt(smt)
```

Comment:
currently the Ally/Ally X are the only known devices that have this issue with SMT. Rather than force this behavior for all devices, could you instead add an `on_suspend` function that will handle SMT if it detects an Ally or Ally X?

You can see an example of how to detect the ally/ally x here: https://github.com/aarron-lee/SimpleDeckyTDP/blob/9382bc135063bea4bcdb95b81379a06eba5743a1/py_modules/cpu_utils.py#L44